### PR TITLE
changelog

### PR DESCRIPTION
**Pull request by tjschuckrug**
Added option to upsample peak in case of Gaussian integration. Relevant for GhOST instrument because of small dwell time values.

### CODE REVIEW

**Comment by Amsoht** [comment]
How urgent is this? do you want me to test this or simply approve it? Note that the master branch is not protected for this repository so you would be able to merge this without any review.

**Comment by Amsoht** [comment]
I am asking, because it would take a couple of days until I would have time to test this.

**Comment by tjschuckrug** [comment]
no urgent request

File: Integration/call_integration.pro [hunk 1]
```diff
@@ -4,26 +4,29 @@
...
-PRO call_integration, sel_chrom, sel_name, PLOT=plot, FIX_XYRANGE=fix_xyrange, MAN_FLAG=man_flag, MAN_COMMENT=man_comment
+PRO call_integration, sel_chrom, sel_name, PLOT=plot, FIX_XYRANGE=fix_xyrange, MAN_FLAG=man_flag, MAN_COMMENT=man_comment, $
+                      gau_upsample=gau_upsample
 
   COMMON DATA
   COMMON COM_PLOT
 
   IF NOT KEYWORD_SET(plot) THEN plot = 0
   IF NOT KEYWORD_SET(fix_xyrange) THEN fix_xyrange = 0
+  ; IF NOT KEYWORD_SET(gau_upsample) THEN gau_upsample = 0
```
Comment (Amsoht) [inline Integration/call_integration.pro]: should this be active?

File: Integration/call_integration.pro [hunk 2]
```diff
@@ -71,6 +74,7 @@ PRO call_integration, sel_chrom, sel_name, PLOT=plot, FIX_XYRANGE=fix_xyrange, M
   IF FINITE(chk_noise) NE 1 THEN chk_noise = 0. ; set zero if not calculated yet
 
   verbose = 0
+  upsample =  1
```
Comment (Amsoht) [inline Integration/call_integration.pro]: should this be removed?

File: Integration/int_gau.pro [hunk 3]
```diff
@@ -25,6 +27,7 @@ FUNCTION int_gau, xval, yval, NSIGMA_FIT=nsigma_fit, NSIGMA_INT=nsigma_int, NTER
   IF timescale EQ 'Minutes' THEN timescale = 60. ELSE timescale = 1. ; 60: minutes / 1: seconds
   nterms = nterms_base + 3 ; nterms paramenter for gaussfit function
   ; datapoints per second? -> gauss min sigma
+  IF NOT keyword_set(upscale) THEN upscale = 0.
```
Comment (Amsoht) [inline Integration/int_gau.pro]: do you mean ´upsample´?

File: Integration/int_gau.pro [hunk 4]
```diff
@@ -35,6 +38,8 @@ FUNCTION int_gau, xval, yval, NSIGMA_FIT=nsigma_fit, NSIGMA_INT=nsigma_int, NTER
       wdth: !VALUES.D_NAN, $
       ts: !VALUES.D_NAN, $
       te: !VALUES.D_NAN, $
+      upscale: 1 ,$
```
Comment (Amsoht) [inline Integration/int_gau.pro]: ´upscale´ is always 1 this way?

Comment (Amsoht) [reply Integration/int_gau.pro]: this gets overwritten by ´scale´ later in l207, I see it's only an initialization

Comment (tjschuckrug) [reply Integration/int_gau.pro]: Yes, this should remain as is, but I will add an explanatory comment

File: Integration/int_gau.pro (hunk 3 above)
Comment (tjschuckrug) [reply Integration/int_gau.pro]: Actually, yes. Oops. Will be changed.

File: Integration/call_integration.pro (hunk 2 above)
Comment (tjschuckrug) [reply Integration/call_integration.pro]: Yes, variable is not needed. This was a line remaining from tests.

**Pull request merged**